.travis.yml

```diff
@@ -1,10 +1,12 @@
 before_install:
+  - gem install bundler
   - gem update bundler
```

Comment:
Aren't those 2 kind of redundant ? I'm pretty sure we could use only one of them.